releasenotes/notes/fix_hellinger_fidelity_def-7f60d53e3577fdaf.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
```

Comment:
I'm not sure this sort of change should be included in a bugfix release

Reply:
It is needed for computing HW metrics, and thus a correct definition would be nice to have sooner than later.

Reply:
If previous definition was incorrect maybe it should be called a bug fix rather than upgrade then?

Reply:
This is done